src/test/java/uk/gov/pay/directdebit/mandate/services/MandateServiceTest.java

```diff
@@ -315,4 +318,16 @@ public void shouldThrowWhenSubmittingTheSameDetailsSeveralTimes_andCreateOnlyOne
         assertThat(newMandate.getState(), is(SUBMITTED));
         verify(mockedDirectDebitEventService, times(1)).registerDirectDebitConfirmedEventFor(newMandate);
     }
+
+    @Test
+    public void mandateExpired_shouldSetStatusToExpired() {
```

Comment:
1) i was wondering if there should be a negative test, as in the MandateService should not be able to expire Mandates that are not AWAITING_DIRECT_DEBIT_DETAILS and SUBMITTED?
2) should we have a test where we expire mandates with states AWAITING_DIRECT_DEBIT_DETAILS and SUBMITTED?

Reply:
Agreed, tests have been added.